sdks/python/hatchet_sdk/config.py

```diff
@@ -64,13 +64,20 @@ class ClientConfig(BaseSettings):
     )
 
     worker_preset_labels: dict[str, str] = Field(default_factory=dict)
+
     enable_force_kill_sync_threads: bool = False
+    enable_thread_pool_monitoring: bool = False
```

Comment:
just to confirm, this automatically gets parsed as an environment variable?

Reply:
yeah, pydantic settings does that for us 🥳

Reply:
so it'll be `HATCHET_CLIENT_ENABLE_THREAD_POOL_MONITORING`